src/org/zaproxy/zap/extension/todo/ZapAddOn.xml

```diff
@@ -0,0 +1,18 @@
+<zapaddon>
+	<name>ToDo-List</name>
+	<version>1</version>
+	<status>alpha</status>
+	<description>Provides a tab which allows you to view the list of test cases</description>
+	<author>ZAP Dev Team</author>
+	<url></url>
+	<changes></changes>
+	<extensions>
+		<extension>org.zaproxy.zap.extension.todo.ExtensionTodoList</extension>
+	</extensions>
+	<ascanrules/>
+	<pscanrules/>
+	<filters/>
+	<files/>
+	<not-before-version></not-before-version>
```

Comment:
We need to set a minimum version, probably 2.6.0 (ZAP does not load older add-ons or add-ons that do not report the minimum version because of incompatibilities).